praw/models/reddit/mixins/replyable.py

```diff
@@ -18,6 +18,10 @@ def reply(self, body):
         comment will be sucessfully created on Reddit and can be retried
         by drawing the comment from the user's comment history.
 
+        .. note:: Some items, such as locked submissions/comments or
+            non-replyable messages will throw an exception when they are tried
```

Comment:
Could we be specific about what said exceptions might be?

Also, consider "will throw an exception when attempting to reply to."

Reply:
re #940 : `prawcore.exceptions.Forbidden`